sentry_sdk/integrations/django/__init__.py

```diff
@@ -232,6 +233,30 @@ def _django_queryset_repr(value, hint):
 _DRF_PATCH_LOCK = threading.Lock()
 
 
+def _patch_db_connect():
+    try:
+        from django.db.backends.base.base import BaseDatabaseWrapper
+    except ImportError:
+        from django.db.backends import BaseDatabaseWrapper
+
+    old_connect = BaseDatabaseWrapper.connect
+
+    def connect(self):
+        hub = Hub.current
+        integration = hub.get_integration(DjangoIntegration)
+        if integration is None:
+            return old_connect(self)
+        else:
+            description = "%s.connect - %s" % (
+                self.settings_dict.get("ENGINE") or "django.db",
+                self.alias,
+            )
+            with hub.start_span(op="db", description=description):
```

Comment:
Can we rename the op to `db.connect`, and move the db engine into tags?

Reply:
will do 👍